### Тема 6 “Понятие о производной”

1. Найти производную выражения:


- a) $\sin{x}⋅\cos{x}$

Решение:

- $\sin{x}⋅\cos{x} = \frac{1}{2}\sin(2x)$  

Производная от $\frac{1}{2}\sin(2x)$:

- $\frac{\mathrm d}{\mathrm d x} \big(\frac{1}{2}\sin(2x)\big) = \frac{1}{2}⋅2\cos(2x) = \cos(2x)$

- b) $\ln(2x+1)^3$

Решение:

Упростим выражение с помощью свойства логарифма:

- $\ln(2x+1)^3 = 3\ln(2x+1)$

Теперь найдем производную:

- $\frac{\mathrm d}{\mathrm d x} \big(3\ln(2x+1)\big) = 3⋅\frac{1}{2x+1}⋅2 = \frac{6}{2x+1}$

- c) $\sqrt{\sin^2(\ln(x^3))}$

Решение:

Упростим выражение:

- $\sqrt{\sin^2(\ln(x^3))} = |\sin(\ln(x^3))|$  

Найдем производную:

- $\frac{\mathrm d}{\mathrm d x} \big(|\sin(\ln(x^3))|\big) = \cos(\ln(x^3))⋅\frac{\mathrm d}{\mathrm d x}\big(\ln(x^3)\big) = \cos(\ln(x^3))⋅\frac{3}{x}$

- d) $\frac{x^4}{\ln(x)}$**  

Решение:

- $\frac{\mathrm d}{\mathrm d x} \big(\frac{x^4}{\ln(x)}\big) = \frac{4x^3⋅\ln(x) - x^4⋅\frac{1}{x}}{(\ln(x))^2} = \frac{4x^3⋅\ln(x) - x^3}{(\ln(x))^2} = \frac{x^3(4\ln(x)-1)}{(\ln(x))^2} $

2. Найти выражение производной функции и ее значение в точке:   

- $f(x) = \cos(x^2+3x)$, $x_0=\sqrt{\pi}$

Решение:

Найдём производную функции:

- $f'(x) = -\sin(x^2+3x)⋅(2x+3)$  

Найдём значение производной в точке $x_0=\sqrt{\pi}$:

- $f'(\sqrt{\pi}) = -\sin((\sqrt{\pi})^2+3\sqrt{\pi})⋅(2\sqrt{\pi}+3) = -\sin(\pi+3\sqrt{\pi})⋅(2\sqrt{\pi}+3)$

In [ ]:
import numpy as np

# Значение точки x0
x0 = np.sqrt(np.pi)

# Вычислим производную в этой точке
u = x0**2 + 3*x0
derivative_value = -np.sin(u) * (2*x0 + 3)

derivative_value


-5.383302410890619

Ответ:

Значение производной функции $f'(x)$ в точке $x_0=\sqrt{\pi}$ равно примерно $-5.383$.

3. Найти значение производной функции в точке:

- $f(x) = \frac{x^3-x^2-x-1}{1+2x+3x^2-4x^3}$, $x_0=0$

Решение:

Найдём производную функции. Используем правило производной частного. Если:
- $f(x) = \frac{u(x)}{v(x)}$, то   
- $f'(x) = \frac{u'(x)v(x) - u(x)v'(x)}{v(x)^2}$  

где:

- $u(x) = x^3-x^2-x-1$  
- $v(x) = 1+2x+3x^2-4x^3$

Теперь найдём производные:

- $u'(x) = 3x^2-2x-1$  
- $v'(x) = 2+6x-12x^2$  

Найдём значение производной в точке $x_0=0$:

- $u(x) = 0^3-0^2-0-1 = -1$  
- $v(x) = 1+2⋅0+3⋅0^2-4⋅0^3 = 1$  
- $u'(x) = 3⋅0^2-2⋅0-1 = -1$  
- $v'(x) = 2+6⋅0-12⋅0^2 = 2$

Подставим все значения в формулу для производной:

- $f'(0) = \frac{(-1)(1)-(-1)(2)}{1^2} = \frac{-1+2}{1} = 1$

Ответ:

Значение производной функции в точке $x_0=0$ равно $1$.

4. Найти угол наклона касательной к графику функции в точке:

- $f(x) = \sqrt{3x}\ln{x}$, $x_0=1$  

Найдём производную функции:

- $f'(x) = \frac{\mathrm d}{\mathrm d x} \big(\sqrt{3x}\big)⋅\ln{x} + \sqrt{3x}⋅\frac{\mathrm d}{\mathrm d x} \big(\ln{x}\big) = \frac{3}{2\sqrt{3x}}⋅\ln{x} + \sqrt{3x}⋅\frac{1}{x}$  

Найдём производную в точке $x_0=1$:

- $f'(1) = \frac{3}{2\sqrt{3⋅1}}⋅\ln{1} + \sqrt{3⋅1}⋅\frac{1}{1} = 0 + \sqrt{3} = \sqrt{3}$  

Угол наклона касательной к графику функции определяется как арктангенс производной в данной точке:

- $θ = \arctan(f'(1)) = \arctan(\sqrt{3})$

Ответ:

$\arctan(\sqrt{3}) = \frac{\pi}{3}$ радиан или 60 градусов, угол наклона касательной в точке $x_0=1$:
- равен $60^∘$.
